src/modules/Servicedomain/Api/Admin.php

```diff
@@ -344,9 +353,20 @@ public function registrar_delete($data)
 
         $model = $this->di['db']->getExistingModelById('TldRegistrar', $data['id'], 'Registrar not found');
 
+        // check if registrar is used by any domain
+        $service_domains = $this->di['db']->find('ServiceDomain', 'tld_registrar_id = ?', ['tld_registrar_id' => $model->id]);
+
+        // Ensure $service_domains is an array before counting its elements
+        $count = is_array($service_domains) ? count($service_domains) : 0; // Handle the case where $service_domains might be null
+
+        if ($count > 0) {
+            throw new \Box_Exception('Registrar is used by :count: domains', ['count' => $count], 707);
```

Comment:
You are using the placeholder wrong here. It needs to be `:count:` in both places, not just one